app/helpers/application_notifications_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ApplicationNotificationsHelper
+  # Returns the view path of the notification
+  #
+  # @param [Course::Notification|UserNotification] notification The target notification
```

Comment:
it doesn't matter if it's new or not, if it's a problem that we identify during a review, it should be fixed.